PROTOCOL.md

```diff
@@ -1170,15 +1171,233 @@ Example:
 }
 ```
 
+### Get Table
```

Comment:
What's the reason for not adding the documentation now?

Reply:
I feel the protocol reflects the expected behavior when user downloads the latest release and start to play with it. Otherwise, the user may issue queries specified by the protocol but don't see expected behavior?

Reply:
Makes sense. We can update the documentation after a release is cut.

Reply:
Done!